src/main/io/osd.c

```diff
@@ -1146,6 +1146,53 @@ static bool osdDrawSingleElement(uint8_t item)
     return true;
 }
 
+#ifdef USE_OSD_STICK_OVERLAY
+static void osdDrawStockOverlayAxis(uint8_t xpos, uint8_t ypos)
```

Comment:
`Stick`?

Reply:
Oops.  Fixed.  Thanks.

Reply:
Not so sure about the array variable, then the user would have to know which index is which parameter.  With distinct names it seems easier.

Reply:
You can define `enum` for characters ... `osdConfig()->overlay[osdStickHorizontal]`

Reply:
Also, the characters will no longer be configurable as discussed above, so fewer parameters now.